pkg/pillar/types/contenttreetypes.go

```diff
@@ -101,17 +101,19 @@ func (config ContentTreeConfig) LogKey() string {
 
 // ContentTreeStatus is response from volumemgr about status of content tree
 type ContentTreeStatus struct {
-	ContentID         uuid.UUID
-	DatastoreID       uuid.UUID
-	DatastoreType     string
-	RelativeURL       string
-	Format            zconfig.Format
-	ContentSha256     string
-	MaxDownloadSize   uint64
-	GenerationCounter int64
-	DisplayName       string
-	HasResolverRef    bool
-	State             SwState
+	ContentID             uuid.UUID
+	DatastoreIDsList      []uuid.UUID
```

Comment:
Nit, but either plural or list not both. IDList is more explicit to me if we also have single datastoreID somewhere.